### DISCUSSION TITLE
karavan-kubernetes, unable to launch apply

### DISCUSSION THREAD

**Discussion by sylim728** [apache/camel-karavan#1241] [Q&A] [answered]
### Ask a question

I download  the karanvan-kubernetes folder and scp into the minikube node
[docs/install/karavan-kubernetes](https://github.com/apache/camel-karavan/tree/main/docs/install/karavan-kubernetes)

The files structure like below: 
![image](https://github.com/apache/camel-karavan/assets/91312395/b444fceb-4b71-4edd-9a44-33b39dc1df11)

When I run the apply command, the error show that no .env file
![image](https://github.com/apache/camel-karavan/assets/91312395/cf8388ae-b1bd-4908-bd4a-512e9be05a1a)

However, I can't find the .env in [docs/install/karavan-kubernetes](https://github.com/apache/camel-karavan/tree/main/docs/install/karavan-kubernetes)

### Variant

None

### Container Management (if applicable)

None

### Operating System (if applicable)

None

### Version

None

**Comment by mgubaidullin** [comment, marked as answer]
![Screenshot_20240423-074303~2](https://github.com/apache/camel-karavan/assets/1379213/937e580a-b9f2-4446-9bdd-862bc53cac4d)

If not **required**, remove or comment this part
